CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+## 7.0.0 (November 19, 2019)
```

Comment:
This is a copy paste from CV file ... version numbers don't even line up with the package.json (this component is currently on 3.1.0, not 7.0.0)